[2022-09-02T00:03:52.512-0400] {plugins_manager.py:292} DEBUG - Plugins are already loaded. Skipping.
[2022-09-02T00:03:52.520-0400] {__init__.py:50} DEBUG - Loading core task runner: StandardTaskRunner
[2022-09-02T00:03:52.529-0400] {taskinstance.py:1107} DEBUG - previous_execution_date was called
[2022-09-02T00:03:52.534-0400] {base_task_runner.py:70} DEBUG - Planning to run as the  user
[2022-09-02T00:03:52.538-0400] {taskinstance.py:871} DEBUG - Refreshing TaskInstance <TaskInstance: test_etl.curate_yahoo_finance_html manual__2022-09-02T04:03:42.436351+00:00 [queued]> from DB
[2022-09-02T00:03:52.549-0400] {taskinstance.py:1181} DEBUG - <TaskInstance: test_etl.curate_yahoo_finance_html manual__2022-09-02T04:03:42.436351+00:00 [queued]> dependency 'Task Instance Not Running' PASSED: True, Task is not in running state.
[2022-09-02T00:03:52.549-0400] {taskinstance.py:1181} DEBUG - <TaskInstance: test_etl.curate_yahoo_finance_html manual__2022-09-02T04:03:42.436351+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2022-09-02T00:03:52.555-0400] {taskinstance.py:1181} DEBUG - <TaskInstance: test_etl.curate_yahoo_finance_html manual__2022-09-02T04:03:42.436351+00:00 [queued]> dependency 'Task Instance State' PASSED: True, Task state queued was valid.
[2022-09-02T00:03:52.556-0400] {taskinstance.py:1181} DEBUG - <TaskInstance: test_etl.curate_yahoo_finance_html manual__2022-09-02T04:03:42.436351+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, The task did not have depends_on_past set.
[2022-09-02T00:03:52.556-0400] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: test_etl.curate_yahoo_finance_html manual__2022-09-02T04:03:42.436351+00:00 [queued]>
[2022-09-02T00:03:52.558-0400] {taskinstance.py:1181} DEBUG - <TaskInstance: test_etl.curate_yahoo_finance_html manual__2022-09-02T04:03:42.436351+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2022-09-02T00:03:52.561-0400] {taskinstance.py:1181} DEBUG - <TaskInstance: test_etl.curate_yahoo_finance_html manual__2022-09-02T04:03:42.436351+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, The task did not have depends_on_past set.
[2022-09-02T00:03:52.565-0400] {taskinstance.py:1181} DEBUG - <TaskInstance: test_etl.curate_yahoo_finance_html manual__2022-09-02T04:03:42.436351+00:00 [queued]> dependency 'Pool Slots Available' PASSED: True, There are enough open slots in default_pool to execute the task
[2022-09-02T00:03:52.565-0400] {taskinstance.py:1181} DEBUG - <TaskInstance: test_etl.curate_yahoo_finance_html manual__2022-09-02T04:03:42.436351+00:00 [queued]> dependency 'Task Concurrency' PASSED: True, Task concurrency is not set.
[2022-09-02T00:03:52.565-0400] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: test_etl.curate_yahoo_finance_html manual__2022-09-02T04:03:42.436351+00:00 [queued]>
[2022-09-02T00:03:52.565-0400] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-09-02T00:03:52.566-0400] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2022-09-02T00:03:52.566-0400] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-09-02T00:03:52.579-0400] {taskinstance.py:1389} INFO - Executing <Task(_PythonDecoratedOperator): curate_yahoo_finance_html> on 2022-09-02 04:03:42.436351+00:00
[2022-09-02T00:03:52.582-0400] {standard_task_runner.py:52} INFO - Started process 20297 to run task
[2022-09-02T00:03:52.589-0400] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'test_etl', 'curate_yahoo_finance_html', 'manual__2022-09-02T04:03:42.436351+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/var/folders/2z/f_7z97sn6_33j1hnl38wlzs40000gn/T/tmpbbf99v7z', '--error-file', '/var/folders/2z/f_7z97sn6_33j1hnl38wlzs40000gn/T/tmp9sry0uqj']
[2022-09-02T00:03:52.591-0400] {standard_task_runner.py:80} INFO - Job 35: Subtask curate_yahoo_finance_html
[2022-09-02T00:03:52.595-0400] {cli_action_loggers.py:66} DEBUG - Calling callbacks: [<function default_action_log at 0x7fc112b283a0>]
[2022-09-02T00:03:52.610-0400] {settings.py:398} DEBUG - Disposing DB connection pool (PID 20297)
[2022-09-02T00:03:52.611-0400] {settings.py:260} DEBUG - Setting up DB connection pool (PID 20297)
[2022-09-02T00:03:52.612-0400] {settings.py:330} DEBUG - settings.prepare_engine_args(): Using NullPool
[2022-09-02T00:03:52.647-0400] {taskinstance.py:1107} DEBUG - previous_execution_date was called
[2022-09-02T00:03:52.655-0400] {task_command.py:371} INFO - Running <TaskInstance: test_etl.curate_yahoo_finance_html manual__2022-09-02T04:03:42.436351+00:00 [running]> on host FAS-EALC-LIU-2
[2022-09-02T00:03:52.656-0400] {taskinstance.py:871} DEBUG - Refreshing TaskInstance <TaskInstance: test_etl.curate_yahoo_finance_html manual__2022-09-02T04:03:42.436351+00:00 [running]> from DB
[2022-09-02T00:03:52.686-0400] {taskinstance.py:1107} DEBUG - previous_execution_date was called
[2022-09-02T00:03:52.690-0400] {taskinstance.py:943} DEBUG - Clearing XCom data
[2022-09-02T00:03:52.752-0400] {taskinstance.py:1581} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=test_etl
AIRFLOW_CTX_TASK_ID=curate_yahoo_finance_html
AIRFLOW_CTX_EXECUTION_DATE=2022-09-02T04:03:42.436351+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-02T04:03:42.436351+00:00
[2022-09-02T00:03:52.753-0400] {__init__.py:146} DEBUG - Preparing lineage inlets and outlets
[2022-09-02T00:03:52.754-0400] {__init__.py:190} DEBUG - inlets: [], outlets: []
[2022-09-02T00:03:53.073-0400] {logging_mixin.py:115} INFO - 
Market price:  126.42 
After hours trading price:  126.41
[2022-09-02T00:03:53.074-0400] {python.py:173} INFO - Done. Returned value was: ('126.42', '126.41')
[2022-09-02T00:03:53.085-0400] {__init__.py:107} DEBUG - Lineage called with inlets: [], outlets: []
[2022-09-02T00:03:53.086-0400] {taskinstance.py:871} DEBUG - Refreshing TaskInstance <TaskInstance: test_etl.curate_yahoo_finance_html manual__2022-09-02T04:03:42.436351+00:00 [running]> from DB
[2022-09-02T00:03:53.094-0400] {taskinstance.py:1418} DEBUG - Clearing next_method and next_kwargs.
[2022-09-02T00:03:53.095-0400] {taskinstance.py:1407} INFO - Marking task as SUCCESS. dag_id=test_etl, task_id=curate_yahoo_finance_html, execution_date=20220902T040342, start_date=20220902T040352, end_date=20220902T040353
[2022-09-02T00:03:53.095-0400] {taskinstance.py:2349} DEBUG - Task Duration set to 0.538068
[2022-09-02T00:03:53.102-0400] {cli_action_loggers.py:84} DEBUG - Calling callbacks: []
[2022-09-02T00:03:53.153-0400] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-02T00:03:53.155-0400] {taskinstance.py:871} DEBUG - Refreshing TaskInstance <TaskInstance: test_etl.curate_yahoo_finance_html manual__2022-09-02T04:03:42.436351+00:00 [running]> from DB
[2022-09-02T00:03:53.179-0400] {dagrun.py:642} DEBUG - number of tis tasks for <DagRun test_etl @ 2022-09-02 04:03:42.436351+00:00: manual__2022-09-02T04:03:42.436351+00:00, state:running, queued_at: 2022-09-02 04:03:42.469689+00:00. externally triggered: True>: 3 task(s)
[2022-09-02T00:03:53.179-0400] {dagrun.py:666} DEBUG - number of scheduleable tasks for <DagRun test_etl @ 2022-09-02 04:03:42.436351+00:00: manual__2022-09-02T04:03:42.436351+00:00, state:running, queued_at: 2022-09-02 04:03:42.469689+00:00. externally triggered: True>: 1 task(s)
[2022-09-02T00:03:53.179-0400] {taskinstance.py:1181} DEBUG - <TaskInstance: test_etl.generate_report manual__2022-09-02T04:03:42.436351+00:00 [None]> dependency 'Previous Dagrun State' PASSED: True, The task did not have depends_on_past set.
[2022-09-02T00:03:53.180-0400] {taskinstance.py:1181} DEBUG - <TaskInstance: test_etl.generate_report manual__2022-09-02T04:03:42.436351+00:00 [None]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2022-09-02T00:03:53.180-0400] {taskinstance.py:1171} DEBUG - Dependencies all met for <TaskInstance: test_etl.generate_report manual__2022-09-02T04:03:42.436351+00:00 [None]>
[2022-09-02T00:03:53.186-0400] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
